core/src/main/res/values/strings.xml

```diff
@@ -376,6 +376,8 @@
     <string name="pref_pauseOnHeadsetDisconnect_title">Headphones Disconnect</string>
     <string name="pref_unpauseOnHeadsetReconnect_title">Headphones Reconnect</string>
     <string name="pref_unpauseOnBluetoothReconnect_title">Bluetooth Reconnect</string>
+    <string name="pref_stream_over_download_title">Show Stream Option on Episode Views</string>
+    <string name="pref_stream_over_download_sum">Episodes displayed in Queue or Podcast View will display a button to stream episode instead of download it.</string>
```

Comment:
I think the wording could be different. I suggest "Prefer streaming", "Display stream button instead of download button in lists."

I think "View" is too technical for average users

Reply:
Applied string changes verbatim